backend/controllers/SiteController.php

```diff
@@ -1,78 +1,12 @@
 <?php
 namespace backend\controllers;
 
-use Yii;
-use yii\filters\AccessControl;
-use yii\web\Controller;
-use common\models\LoginForm;
-use yii\filters\VerbFilter;
+use common\controllers\UserController;
 
 /**
  * Site controller
  */
-class SiteController extends Controller
+class SiteController extends UserController
```

Comment:
not the best idea IMO
why SiteController should inherit all the functionality from the UserController ?
They have to be separated.